scripts/run_test_in_devicefarm.sh

```diff
@@ -67,14 +67,15 @@ while true; do
   read -a result_arr <<< $run_status_response
   status=${result_arr[0]}
   result=${result_arr[1]}
-  echo "Status = $status Result = $result"
   if [ "$status" = "COMPLETED" ]
   then
     break
   fi
   sleep 5
 done
+echo "Status = $status Result = $result"
 
+./scripts/generate_df_testrun_report --run_arn="$run_arn" --module_name="$module_name" --pr="$CODEBUILD_SOURCE_VERSION" --output_path="build/allTests/$module_name/"
```

Comment:
Will be easier to read like this:

```suggestion
./scripts/generate_df_testrun_report \
    --run_arn="$run_arn" \
    --module_name="$module_name" \
    --pr="$CODEBUILD_SOURCE_VERSION" \
    --output_path="build/allTests/$module_name/"
```